src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+server:
+  port: 8080
+spring:
```

Comment:
What happens if we want to escalate this service?, we would have a database for each service, how would the info be synchronized?